.github/scripts/end2end/configure-e2e-ctst.sh

```diff
@@ -18,6 +18,12 @@ UUID=${UUID:1}
 OPLOG_TOPIC="${UUID}.backbeat-oplog"
 NOTIFICATION_TOPIC="${UUID}.backbeat-notification"
 
+kubectl patch zenko ${ZENKO_NAME} --type merge -p '{"spec":{"backbeat":{"triggerExpirationsOneDayEarlierForTesting": false}}}'
+kubectl patch zenko ${ZENKO_NAME} --type merge -p '{"spec":{"backbeat":{"lifecycleBucketProcessor": {"triggerTransitionsOneDayEarlierForTesting" : false}}}}'
```

Comment:
instead of deploying zenko, then patching : can we instead "parametrize" the Zenko CR (with different values for these fields in the different branches of the workflow), so that we directly deploy the right zenko?

this will be faster than deploying & reconfiguring (which may even happen multiple times, since we are patching the CR 3 times)